### PR TITLE
accesslog: add field to TLSProperties in data.accesslog.v3.AccessLogCommon

### PR DESCRIPTION
**Pull request by liwanxue**
Commit Message: accesslog: add fields TLSProperties in data.accesslog.v3.AccessLogCommon
Additional Description: The substitution formatter offers this field, but they were not present in the ALS proto. The patch plumbs in the the same values read in substitution_formatter.cc ([e.g.](https://github.com/envoyproxy/envoy/blob/f38a9e13e67b0e2506a0ddb20239eaea86e9b554/source/common/formatter/substitution_formatter.cc#L1462-L1469)) into new fields in the access log proto. The doc strings are copy/pasted from the existing docs ([e.g.](https://github.com/envoyproxy/envoy/blob/e485fa5a995fb1de1efb34d0e5bbb05b27325fbd/docs/root/configuration/observability/access_log/usage.rst?plain=1#L1005-L1009)).
Risk Level: Low
Testing: The patch contains minimal test changes, just what was needed to get existing integration tests to pass.
Docs Changes: Yes, API docs in proto.
Release Notes: Yes.
Platform Specific Features: N/A
API Considerations: Added new fields to ALS proto.

### CODE REVIEW

**Comment by wbpcode** [comment]
Could you check the CI first? Thanks.

/wait

File: source/extensions/access_loggers/grpc/grpc_access_log_utils.cc [hunk 1]
```diff
@@ -203,7 +203,9 @@ void Utility::extractCommonAccessLogProperties(
       peer_san->set_uri(MessageUtil::sanitizeUtf8String(uri_san));
     }
 
-    peer_properties->set_subject(downstream_ssl_connection->subjectPeerCertificate());
+    peer_properties->set_subject(MessageUtil::sanitizeUtf8String(downstream_ssl_connection->subjectPeerCertificate()));
```
Comment (wbpcode) [inline source/extensions/access_loggers/grpc/grpc_access_log_utils.cc:206]: This sanitization like a very minor behavior change. Is this necessary?

Comment (liwanxue) [reply source/extensions/access_loggers/grpc/grpc_access_log_utils.cc]: removed.

**Comment by liwanxue** [comment]
@wbpcode  may you please help check PR?  I fixed few CI failures, but still others failures in CI, I have no idea to fix, may you please provide some suggestions?

**Comment by liwanxue** [comment]
this is part of https://github.com/envoyproxy/envoy/issues/31636

**Comment by wbpcode** [comment]
image

File: changelogs/current.yaml [hunk 2]
```diff
@@ -36,6 +36,9 @@ behavior_changes:
     Handle empty response bodies in grpc_http1_reverse_bridge. This may cause problems for clients expecting the filter to crash
     for empty responses.  This behavioral change can be temporarily reverted by setting runtime guard
     ``envoy.reloadable_features.grpc_http1_reverse_bridge_handle_empty_response`` to false.
+- area: access_log
+  change: |
+    added issuer in certificate_properties to the gRPC access log service(AlS).
```
Comment (wbpcode) [inline changelogs/current.yaml:41]:
```suggestion
    Added issuer in certificate_properties to the gRPC access log service(AlS).
```

**Review by wbpcode — COMMENTED** [review]
You can accept the suggestions and give the ci another try.

**Review by wbpcode — APPROVED** [review]
LGTM. Thanks.

**Comment by liwanxue** [comment]
@wbpcode  I fixed some tests, may you pls approve again? Thanks!

**Review by wbpcode — APPROVED** [review]
LGTM. Thanks.

**Comment by wbpcode** [comment]
Sorry for the delay. Let's waiting the CI to pass.

**Comment by liwanxue** [comment]
> LGTM. Thanks.

I am sorry, just push a commit to make PR clean, please approve it again= =

**Comment by wbpcode** [comment]
@liwanxue force push is unfavorable basically. And all commit will be squashed, you needn't to do the commit clean.

**Review by wbpcode — APPROVED** [review]
LGTM. Thanks.

**Comment by wbpcode** [comment]
Could you just merge main and kick the CI? Seems it's broken.

**Comment by liwanxue** [comment]
@wbpcode  may you please help cherry-pick this PR to v1.28? or shall I do anything to get this PR cherry-pick to v1.28?

**Pull request merged**